x-pack/plugins/task_manager/server/task.ts

```diff
@@ -277,6 +277,11 @@ export interface TaskInstance {
    * The random uuid of the Kibana instance which claimed ownership of the task last
    */
   ownerId?: string | null;
+
+  /**
+   * Indicates whether the task is currently enabled. Disabled tasks will not be claimed.
+   */
+  enabled?: boolean;
```

Comment:
Yes, we need to do a partial update in the task manager task runner without `enabled` to avoid overwriting any user-initiated changes to the `enabled` status. Alternatively we could `get` the task document again before updating to make sure we have the latest `enabled` value, but that would mean adding 2 additional `gets` to the task running process (once during `markAsRunning` and once after the run finishes.

Reply:
Ya, we do *NOT* want to do more GETs :-) - partial update is fine.